### PR TITLE
1.2.5 contracts

### PR DESCRIPTION
**Pull request by wilbarnes**
univ2daiusdc & univ2ethusdt lp tokens

### CODE REVIEW

File: releases/mainnet/1.2.5/index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	</head>
+	<body>
+		<div>
+			<h2>
+				<strong>Mainnet Release 1.2.5</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/mainnet/1.2.5/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmWGjaY1pacjp9T8JAQ9VbrJWBsgsNzgvwduDccr1gMjUq">Contract addresses (IPFS hosted)</a></li>
```
Comment (zdumitru) [inline releases/mainnet/1.2.5/index.html:13]: This ipfs JSON doesn't match releases/mainnet/1.2.5/contracts.json  file.

**Review by zdumitru — CHANGES_REQUESTED** [review]
IPFS file needs to be changed. Also please update main index.html file with the links to mainnet pages.

File: releases/mainnet/1.2.5/index.html (hunk 1 above)
Comment (brianmcmichael) [reply releases/mainnet/1.2.5/index.html:13]: I'll take care of this.

Comment (brianmcmichael) [reply releases/mainnet/1.2.5/index.html]: added

**Review by zdumitru — CHANGES_REQUESTED** [review]

File: releases/mainnet/active [hunk 2]
```diff
@@ -1 +1 @@
-1.2.4
\ No newline at end of file
+1.2.5
```
[the remark this answers is not in the document]
Comment (brianmcmichael) [reply releases/mainnet/active:1]: Do you do that in a separate PR? I can revert this.

[the remark this answers is not in the document]
Comment (brianmcmichael) [reply releases/mainnet/active]: changed back to 1.2.4

**Comment by zdumitru** [comment]
Table in the main index.html needs to be updated to include link to release 1.2.5

**Review by zdumitru — CHANGES_REQUESTED** [review]
@wilbarnes Please add 1.2.5 link [on top of this line](https://github.com/makerdao/mcd-changelog/blob/master/index.html#L54) and then it's good to be merged.

File: releases/mainnet/1.2.5/index.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+				<strong>Mainnet Release 1.2.5</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/mainnet/1.2.5/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmbVHXvPGPBNJoHLmaJ8CkYTVWXic7EextRvnkgRDLTCMa">Contract addresses (IPFS hosted)</a></li>
+				<li><a href="/releases/mainnet/1.2.5/abi/index.html">ABIs</a></li>
+				<!-- <li>Links to GitHub repositories</li>
+					<ul>
+						DSS deploy scripts (<a href="https://github.com/makerdao/dss-deploy-scripts/releases/tag/1.2.5" target="_blank">changelog</a>)</li>
```
Comment (gbalabasquer) [inline releases/mainnet/1.2.5/index.html:17]: This can be uncommented now: https://github.com/makerdao/dss-deploy-scripts/releases/tag/1.2.5

Comment (wilbarnes) [reply releases/mainnet/1.2.5/index.html:17]: done

**Comment by wilbarnes** [comment]
added 1.2.5 link to index.html, should be g2g now

**Review by zdumitru — APPROVED** [review]

**Pull request merged**